apps-rendering/src/capi.ts

```diff
@@ -214,6 +215,31 @@ const capiDateTimeToDate = (date: CapiDateTime): Option<Date> =>
 const maybeCapiDate = (date: CapiDateTime | undefined): Option<Date> =>
 	pipe(date, fromNullable, andThen(capiDateTimeToDate));
 
+/**
+ * Return a mock `Newsletter` if `content` contains a newsletter tag
+ * @param content the content to check for presence of a newsletter tag
+ * @returns a mock `Newsletter`, or `undefined` if `content` does not include a newsletter tag
+ */
+const getMockPromotedNewsletter = (
+	content: Content,
+): Newsletter | undefined => {
+	const newsletterTagPrefix = 'campaign/email/';
+	const hasNewsletterTag = tagsOfType(TagType.CAMPAIGN)(content.tags).some(
+		(campaignTag) => campaignTag.id.startsWith(newsletterTagPrefix),
+	);
+
+	if (hasNewsletterTag) {
+		return {
+			description: 'Test newsletter',
+			frequency: 'test',
+			identityName: 'invalid',
+			name: 'Test newsletter',
+			successDescription: 'test',
+			theme: 'news',
+		};
+	}
+};
+
```

Comment:
Not essential, but it might be useful to include the tagId in the mock Newsletter's name to indicate which newsletter the article is tagged with. 

```suggestion
	const newsletterTag = tagsOfType(TagType.CAMPAIGN)(content.tags).find(
		(campaignTag) => campaignTag.id.startsWith(newsletterTagPrefix),
	);

	if (newsletterTag) {
		return {
			description: 'Test newsletter',
			frequency: 'test',
			identityName: 'invalid',
			name: `Test: ${newsletterTag.id}`,
			successDescription: 'test',
			theme: 'news',
		};
	}
};

```